src/content/configuration/resolve.mdx

```diff
@@ -731,15 +731,34 @@ module.exports = {
 };
 ```
 
-A regular expression, or an array of regular expressions, can be used to test file paths and only cache certain modules. For example, to only cache utilities:
+When an object is provided, webpack will use it as cache.
+
+For example, you can supply a [Proxy object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) instead of regular one:
```

Comment:
```suggestion
For example, you can supply a [Proxy object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) instead of a regular one:
```